07_control_statements_branching_and_jumps/exercises/switch.c

```diff
@@ -0,0 +1,31 @@
+// Redo exercise 4 using a switch.
+#include <stdio.h>
+
+int main(void)
+{
+    char ch;
+    int point_count = 0;
+    int excl_mark_count = 0;
+
+    while ((ch = getchar()) != '#')
+    {
+        switch (ch)
```

Comment:
Let's keep switch formatting like in the book

```suggestion
        switch (ch)
        {
            case '.': putchar('!');
                      point_count++;
                      break; 
            ...
```